tests/std/tests/P0355R7_calendars_and_time_zones_time_zones/test.cpp

```diff
@@ -91,9 +117,268 @@ void timezone_names_test() {
     assert(_Locate_zone_impl(tzdb.zones, "America/Nuuk") == nullptr); // does not exist in ICU (very rare)
 }
 
+void validate_timezone_transitions(const time_zone* tz, const Transition& data) {
+    auto info = tz->get_info(data.begin());
+    assert(info.begin == data.begin());
+    assert(info.end == data.end());
+    assert(info.offset == data.offset());
+    assert(info.save == data.save());
+    assert(info.abbrev == data.abbrev());
+}
+
+void timezone_sys_info_test() {
+    const auto& tzdb = get_tzdb();

```

Comment:
`tzdb` is the name of a `std::chrono` type; I strongly recommend another name for this reference. Occurs below.